pilota-build/test_data/thrift_workspace_with_split/output/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[workspace]
+members = [
+    "article",
+    "author", "common",
```

Comment:
When I use the `UPDATE_TEST_DATE=1 cargo test` command, the common in members will disappear. Does it the same for you? @missingdays